01week/rockPaperScissors.js

```diff
@@ -11,9 +11,86 @@ const rl = readline.createInterface({
 function rockPaperScissors(hand1, hand2) {
 
   // Write code here
+//hand 1 enters their weapon, hand 2 enters their weapon - This is done
 
+if(typeof(hand1)=='string' && typeof(hand2)=='string'){
+} else {
```

Comment:
This indentation needs to be fixed, this isn't readable. Also format the if/else statement needs to be cleaned up 
`if (typeof(hand1)!='string' && typeof(hand2)!='string) return 'Invalid entry, please choose rock, paper, or scissors'`. Console logging a string will not end the function, in addition, the prompt function console logs the return of this function